ci-operator/step-registry/ipi/conf/vsphere/mirror-image/ipi-conf-vsphere-mirror-image-commands.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+mirror_registry_url="vmc.mirror-registry.qe.devcluster.openshift.com:5000"
```

Comment:
Can we also move this url to vault to avoid hard code, also help us quick switch over a different url when needed.

Reply:
sure, updated, related job is passed.